logos/innovation_growth_hub/innovation_growth_hub.svg

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 1080 1080" style="enable-background:new 0 0 1080 1080;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#3A4195;}
+	.st1{fill:#A9CE43;}
+	.st2{fill:#F58733;}
+	.st3{fill:#23ADE4;}
+	.st4{fill:#6E508D;}
+</style>
+<image style="display:none;overflow:visible;" width="1107" height="651" id="XMLID_2_" xlink:href="../../Innovation Growth Hub logo x.jpg"  transform="matrix(0.7989 0 0 0.7989 97.7945 238.0172)">

```

Comment:
Hey, nice! @UfereSamuel 
It seems, there is an image here. Please take out the images and push a new commit with just pure SVG.
Don't forget to optimise your SVG & PNG with the tools mentioned here in [contributing.md](https://github.com/PaystackHQ/nigerialogos/blob/master/contributing.md) before pushing your new commit